libfdisk/meson.build

```diff
@@ -56,15 +56,7 @@ lib__fdisk = static_library(
   include_directories : lib_fdisk_includes,
   dependencies : build_libfdisk ? [] : disabler())
 
-lib_fdisk_static = static_library(
-  'fdisk_static',
-  link_whole : lib__fdisk,
-  link_with : [lib_common,
-               lib_uuid.get_static_lib()],
-  dependencies : [blkid_static_dep],
-  install : false)
-
-lib_fdisk = library(
+lib_fdisk = both_libraries(
```

Comment:
This is wrong, since it will now start to install both of them when before it only installed whichever was the value of default_library.

Reply:
Hrm seems so:

```
> find /usr/lib64/ | grep fdisk
/usr/lib64/libfdisk.so.1
/usr/lib64/libfdisk.so.1.1.0
```

Reply:
Removed.

Reply:
Perhaps I am missing the part where you removed it, because I don't actually see where you did any such thing?

Reply:
I added it back.